inspire/modules/deposit/tasks.py

```diff
@@ -34,6 +34,26 @@
 
 
 #
+# Helper functions
+#
+def save_deposition_file(obj, deposition, filename, file_path, log_messages={}):
```

Comment:
Great that you factored this part out as a helper function. :+1: 

However, I will suggest to not couple this function to the `obj` object. I see that this is used for logging, this can rather happen on the caller side. Then it can `raise` in the except block instead of logging. 

In the callers side it could look like:
```python
try:
    save_deposition_file(deposition, filename, file_path)
except FilenameAlreadyExists as err:
    obj.log.error(str(err))
```